app/components/Intercept_Components/index.tsx

```diff
@@ -1,23 +1,18 @@
 import * as React from "react";
-import {InterceptTextBox} from "./InterceptTextBox";
-import {RequestObj} from "./../request_list";
+import { InterceptTextBox } from "./InterceptTextBox";
+import { RequestObj } from "./../request_list";
 
-
-export const InterceptForm:React.SFC<RequestObj> = (props) => {
+export const InterceptForm: React.SFC<RequestObj> = props => {
   return (
     <div>
       <InterceptTextBox
+        data={props.data}
```

Comment:
Why this component need whole data? This is per row and can be should accept only the fields it need for that particular row.